calendar-tool/tool/src/java/org/sakaiproject/calendar/entityproviders/CalendarEventEntityProvider.java

```diff
@@ -191,32 +176,141 @@ public boolean entityExists(final String id) {
 	 * event/siteId/eventId
 	 */
 	@EntityCustomAction(action = "event", viewKey = EntityView.VIEW_LIST)
-	public CalendarEventDetails getCalendarEventDetails(EntityView view) {
+	public CalendarEventDetails getCalendarEventDetails(final EntityView view) {
 		// get siteId
-		String siteId = view.getPathSegment(2);
-		String eventId = view.getPathSegment(3);
+		final String siteId = view.getPathSegment(2);
```

Comment:
new class in sakai trunk has this change so I kept my added code in line with that.